docs/releases/1.11.txt

```diff
@@ -313,6 +313,9 @@ Models
 * You can now use the ``unique=True`` option with
   :class:`~django.db.models.FileField`.
 
+* Fixed :class:`~django.db.models.Prefetch` objects to pickle without
```

Comment:
This looks like a bug fix (which isn't included in major version release notes) rather than a minor feature.